src/test/java/org/gridsuite/dynamicmargincalculation/server/service/DynamicMarginCalculationResultServiceTest.java

```diff
@@ -0,0 +1,112 @@
+/*
+ * Copyright (c) 2025, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+package org.gridsuite.dynamicmargincalculation.server.service;
+
+import org.gridsuite.dynamicmargincalculation.server.DynamicMarginCalculationException;
+import org.gridsuite.dynamicmargincalculation.server.dto.DynamicMarginCalculationStatus;
+import org.gridsuite.dynamicmargincalculation.server.entities.DynamicMarginCalculationResultEntity;
+import org.gridsuite.dynamicmargincalculation.server.repositories.DynamicMarginCalculationResultRepository;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.Test;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+
+import java.util.List;
+import java.util.Optional;
+import java.util.UUID;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+/**
+ * @author Thang PHAM <quyet-thang.pham at rte-france.com>
+ */
+@SpringBootTest
+class DynamicMarginCalculationResultServiceTest {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(DynamicMarginCalculationResultServiceTest.class);
+
+    @Autowired
+    DynamicMarginCalculationResultRepository resultRepository;
+
+    @Autowired
+    DynamicMarginCalculationResultService dynamicSecurityAnalysisResultService;
+
+    @AfterEach
+    void cleanDB() {
+        resultRepository.deleteAll();
+    }
+
+    @Test
+    void testCrud() {
```

Comment:
It's heavy to add all those tests for an "empty" server. You could make it simpler before you really code into your DMC server

Reply:
MarginCalculationResult contains a list of LoadIncreaseResult in which we have a status. For this reason, it is better to separate gridsuite status into a separated table (as we done in LoadFlow)
I have updated my PR with the commit here https://github.com/gridsuite/dynamic-margin-calculation-server/pull/1/commits/75e610f0dc7b71a9345f0779ff23c228c63738fc